docs/readmes/proposals/p024_magma_settlement_service.md

```diff
@@ -0,0 +1,280 @@
+
+## 3.0 Implementation scope
+
+### 3.1 High - level solution architecture
+
+![arch](https://user-images.githubusercontent.com/93994458/149513109-47b80417-7488-49c4-a2f0-ff94e6c57ebf.png)
+
+### 3.2 End to end scenarios
+
+Below will be described e2e scenarios needs to be implemented in scope of this project (Note: scenarios may be decomposed into different use-cases depends on actors involved and/or business case):
+
+- State channel management procedure
+
+The procedure describes Opening, Update and Closure of State channel management call flow.
+The detailed call flow for the procedure is specified in section 3.2.1 “State channel management procedure”.
+
+- UE attach procedure
+
+The procedure in which the UE registers to the network, and receives the usage quota to create the EPS Bearer between the UE and the PGW, in order to be able to send and receive data.
+The detailed call flow for the procedure is specified in section 3.2.2 “UE attach procedure”.
+
+- UE quota update procedure
+
+The procedure describes a next quota request by Threshold from the previously received quota from SGW to OCS.
+The detailed call flow for the procedure is specified in section 3.2.3 “UE quota update procedure”.
+
+- UE initiated session termination procedure
+
+The procedure in which the UE initiates session termination. The procedure includes the last quota usage report from SGW to OCS.
+The detailed call flow for the procedure is specified in section 3.2.4 “UE initiated session termination procedure”.
+
+- Network initiated termination procedure
+
+The procedure in which the Network (S/PGW) initiates session termination. The procedure includes the last quota usage report from SGW to OCS.
+The detailed call flow for the procedure is specified in section 3.2.5 “Network initiated termination procedure”.
+
+- TAP-Out generation and exporting procedure
+
+The procedure to generate TAP3 files and transfer the files to the DCH (HPLMN) partner.
+
+- RAP processing procedure
+
+The procedure to return rejected TAP files and records to the visited network operator for corrections.
+
+#### 3.2.1 State channel management procedure
+
+Following diagram shows e2e State Channel life cycle management that will be in real life. It consists of existing functionality (marked with black), new functionality to be implemented in magma (marked with blue).
+
+![State channel management procedure (1)](https://user-images.githubusercontent.com/93994458/149513469-0380c35c-d109-4c27-8014-d05b6cbd9a5e.png)
+
+This scenario consists of the steps below:
+
+1. Step [1-2] The State channel opening procedure. The Helium Handler sends Open State Channel Request to open the State Channel. State Channel is opened once the positive Response is received from State Channel.
```

Comment:
Here and elsewhere. Maybe use bulleted list here instead of ordered list, as it's a bit confusing with the steps numbered 1-5 and bullets 1-3?

Reply:
agree. done